k8s/namespaces/pcq/pcq-consolidation-service/pcq-consolidation-service.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-consolidation-service
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:prod-*
```

Comment:
```suggestion
  annotations:
    hmcts.github.com/prod-automated: enabled
```

Reply:
Thanks @adusumillipraveen your suggestions have been implemented and I've removed the common equivilents.